docs/components/TryDemo.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div class="try-out-wrapper">
+        <slot></slot>
+        <a :href="`https://dashboard-demos.flowfuse.cloud/dashboard/${href}`" target="_blank" class="--btn">
+            {{ title }}
```

Comment:
Would be good to have an External Link icon in htis button to indicate users will navigate away from the docs

Reply:
<img width="782" alt="Screenshot 2024-07-26 at 22 14 52" src="https://github.com/user-attachments/assets/cda25a42-8d9f-49c3-8cc4-98a4c3a0f571">

Added the SVG icon from material icons as the Vuetify is only supports Dashboard at the moment